src/rest-server/controllers/monthlyDonation.js

```diff
@@ -29,7 +40,10 @@ const monthlyDonation = {
             items: [{ plan: "plan_Ds6HYG3GMu3nnm" }]
```

Comment:
You can test this with the plan ID of "plan_Ds3ZmQIVpmhJl4" and only select the $25 when you test stripe